checks/ci_tests.go

```diff
@@ -43,19 +44,18 @@ func init() {
 
 // CITests runs CI-Tests check.
 func CITests(c *checker.CheckRequest) checker.CheckResult {
-	prs, _, err := c.Client.PullRequests.List(c.Ctx, c.Owner, c.Repo, &github.PullRequestListOptions{
-		State: "closed",
-	})
+	prs, err := c.RepoClient.ListMergedPRs()
 	if err != nil {
-		e := sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("Client.PullRequests.List: %v", err))
+		e := sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("RepoClient.ListMergedPRs: %v", err))
 		return checker.CreateRuntimeErrorResult(CheckCITests, e)
 	}
 
 	usedSystem := unknown
 	totalMerged := 0
 	totalTested := 0
-	for _, pr := range prs {
-		if pr.MergedAt == nil {
+	for index := range prs {
+		pr := &prs[index]
```

Comment:
Any reason in particular for doing this (as opposed to how we were doing it before)? To avoid copying?

Reply:
Yeah, go-linter wasn't happy with `pr` struct being copied every loop.